src/test/java/automationpractice/com/pages/AddressCheckoutPage.java

```diff
@@ -0,0 +1,79 @@
+package automationpractice.com.pages;
+
+import automationpractice.com.domain.NewAddressData;
+import net.serenitybdd.core.pages.PageObject;
+import net.serenitybdd.core.pages.WebElementFacade;
+import org.openqa.selenium.By;
+import org.openqa.selenium.support.FindBy;
+
+public class AddressCheckoutPage extends PageObject {
+
+    @FindBy(name = "processAddress")
+    private WebElementFacade clickOnProceedToCheckoutButtonInAddressLabel;
+
+    @FindBy(xpath = "//*[contains(@class,'address_add submit')]//*[contains(@class,'button')]")
+    private WebElementFacade clickAddANewAddressButton;
+
+    @FindBy(xpath = "//*[@id='address_delivery']//*[@class='address_firstname address_lastname']")
+    private WebElementFacade nameAndLastNameInDeliveryAddress;
+
+    @FindBy(xpath = "//*[@id='address_delivery']//*[@class='address_city address_state_name address_postcode']")
+    private WebElementFacade cityAndStateAndZipCodeInDeliveryAddress;
+
+    @FindBy(xpath = "//*[@id='center_column']//*[@class='checkbox addressesAreEquals']")
+    private WebElementFacade clickDisagreeUseTheDeliveryAddressAsTheBillingAddress;
+
+    @FindBy(xpath = "//*[@id='address_invoice']//*[@class='address_firstname address_lastname']")
+    private WebElementFacade nameAndLastNameInBillingAddress;
+
+    @FindBy(xpath = "//*[@id='address_invoice']//*[@class='address_city address_state_name address_postcode']")
+    private WebElementFacade cityAndStateAndZipCodeInBillingAddress;
+
+    @FindBy(xpath = "//*[@id='ordermsg']//*[@class='form-control']")
+    private WebElementFacade addCommentInAddressTab;
```

Comment:
1. //*[contains(@class,'address_add submit')] - jeśli ktoś tutaj np zmieni kolejność klas z address_add submit na submit address_add ten xpath juz nie przejdzie. Lepiej gdyby to wyglądało w ten sposób "//*[contains(@class,'address_add') and contains(@class, 'submit')]"
2. Podobny problem z pozostałymi xpathami.